web/src/main/java/com/bazaarvoice/emodb/web/EmoModule.java

```diff
@@ -699,6 +710,16 @@ protected void configure() {
             bind(IpBlacklistControlTask.class).asEagerSingleton();
             bind(AdHocThrottleControlTask.class).asEagerSingleton();
             bind(AdHocThrottleManager.class).asEagerSingleton();
+            bind(DataStoreUpdateThrottleManager.class).asEagerSingleton();
```

Comment:
Can't because `DataStoreUpdateThrottleManager` is also injected to the task [here](https://github.com/bazaarvoice/emodb/pull/196/files#diff-77bec16c35bda425709b853bcb85f1a9R74).